phpunit/class-block-library-comment-template-test.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Tests server side rendering of core/comment-template
+ *
+ * @package    Gutenberg
+ * @subpackage block-library
+ */
+
+/**
+ * Tests for rendering a list of comments
+ */
+class Block_Library_Comment_Template_Test extends WP_UnitTestCase {
+
+	private static $custom_post;
+	private static $comment_ids;
+
+	public function setUp() {
+
+		self::$custom_post = self::factory()->post->create_and_get(
+			array(
+				'post_type'    => 'dogs',
+				'post_status'  => 'publish',
+				'post_name'    => 'metaldog',
+				'post_title'   => 'Metal Dog',
+				'post_content' => 'Metal Dog content',
+				'post_excerpt' => 'Metal Dog',
+			)
+		);
+
+		self::$comment_ids = self::factory()->comment->create_post_comments(
+			self::$custom_post->ID,
+			1,
+			array(
+				'comment_author'       => 'Test',
+				'comment_author_email' => 'test@example.org',
+				'comment_content'      => 'Hello world',
+			)
+		);
+
+	}
+
+	public function tearDown() {
+		wp_delete_post( self::$custom_post );
```

Comment:
I'm wondering if we really need this `tearDown` call. I can't see any cleanup in the existing unit tests for comments. Maybe it's handled by the test framework? See an example:

https://github.com/WordPress/wordpress-develop/blob/2648a5f984b8abf06872151898e3a61d3458a628/tests/phpunit/tests/comment/template.php

Reply:
I was thinking that the database needs to be cleaned out after each test or test suite so that it does not affect other tests, but perhaps I was wrong. I ll check it 👍

Reply:
I would expect the cleanup as well, but I checked 3 test suites and they didn’t perform the cleanup. It made me think it might be already embeded in the exposed API.

Reply:
Done in 4913a8c